content/doc/book/pipeline/syntax.adoc

```diff
@@ -1278,7 +1277,7 @@ Groovy learning-curve isn't typically desirable for all members of a given
 team, so Declarative Pipeline was created to offer a simpler and more
 opinionated syntax for authoring Jenkins Pipeline.
 
-The two are both fundamentally the same Pipeline sub-system underneath. They
+Both are fundamentally the same Pipeline sub-system underneath. They
```

Comment:
I would spell this out here: "Scripted and Declarative sit on top of the same Pipeline subsystem.
Both use a `Jenkinsfile` and are durable implementations of "Pipeline as code." 
Both use steps that are either built into Pipeline or provided by plugins.
Both can utilize <<shared-libraries#, Shared Libraries>><<shared-libraries#, Shared Libraries>>.

I might consider a bullet list for these characteristics.